From bdb818d42bc99ab4b273e6741bb8dedc29cc0477 Mon Sep 17 00:00:00 2001
From: Andrei Catalin Ilie <andreicatalin.ilie@nxp.com>
Date: Fri, 27 Sep 2019 13:00:29 +0300
Subject: [PATCH 5/5] Add build script for i.MX8 target

This script must be modified to use the toolchain from the latest SDK.
---
 build_script.sh | 23 +++++++++++++++++++++++
 1 file changed, 23 insertions(+)
 create mode 100644 build_script.sh

diff --git a/build_script.sh b/build_script.sh
new file mode 100644
index 0000000..63ce3b8
--- /dev/null
+++ b/build_script.sh
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+mkdir build/
+source /opt/fsl-imx-internal-xwayland/4.19-thud/environment-setup-aarch64-poky-linux
+cd build
+
+cmake \
+              -DCMAKE_BUILD_TYPE=Release	\
+ 	      -DENABLE_PYRENDERDOC=OFF		\
+              -DENABLE_QRENDERDOC=OFF		\
+	      -DENABLE_RENDERDOCCMD=ON		\
+              -DENABLE_XLIB=OFF			\
+              -DENABLE_XCB=OFF			\
+              -DENABLE_GL=OFF			\
+	      -DENABLE_GLES=OFF			\
+	      -DENABLE_VULKAN=ON		\
+	      -DENABLE_WAYLAND=ON		\
+	      -DUSE_INTERCEPTOR_LIB=OFF		\
+	      -Bbuild -H.			\
+              -DCMAKE_TOOLCHAIN_FILE=/opt/fsl-imx-internal-xwayland/4.19-thud/sysroots/x86_64-pokysdk-linux/usr/share/cmake/OEToolchainConfig.cmake \
+..
+
+make -j 4
-- 
2.7.4

